data/settings.js

```diff
@@ -1068,13 +1220,13 @@ Value format: \`<header-field> [ ":" <regexp> ]\`
 Some spam scanners include the maximum score value in one of their status
```

Comment:
sieve_spamtest_score_max_header default should be removed.

Reply:
fixed